e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -135,6 +138,11 @@ func InitializeIngestManagerTestSuite(ctx *godog.TestSuiteContext) {
 			"stackVersion":  common.StackVersion,
 		}
 
+		if !shell.GetEnvBool("SKIP_PULL") {
```

Comment:
There is a section for `Environment variables affecting the build`: https://github.com/elastic/e2e-testing/tree/master/e2e#environment-variables-affecting-the-build Can we add info about this new var there?